app/package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "cross-env CI=true react-scripts test --env=jsdom",
+    "test": "cross-env CI=true react-scripts test --env=jsdom --passWithNoTests",
```

Comment:
Is this:
1. given no tests, pass anyway
or
2. always pass, even if tests fail

Reply:
It is case 1, it passes even though the codebase contains no tests, so this is perfectly fine.